pkg/services/folder/model.go

```diff
@@ -56,6 +56,7 @@ type UpdateFolderCommand struct {
 	NewUID         *string `json:"uid"`
 	NewTitle       *string `json:"title"`
 	NewDescription *string `json:"description"`
+	ExistingUid    string  `json:"existinguid"`
```

Comment:
is it ok to add this? i need to pass an existing uid in pkg/services/folder/folderimpl/folder.go on ln 217 but i think i'm missing something somewhere

Reply:
The existing folder is in the `Folder` field and you can get its UID from that.

Reply:
perfect, thanks!